EIPS/eip-3372.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: 3372
+title: Apply minor modifications to the Ethash algorithm to break current ASIC implementations
+status: Draft
+type: Standards Track
+category: Core
+author: mineruniter969 (@mineruniter969), mineruniter969 <mineruniter969@tutanota.com>
+discussions-to: https://ethereum-magicians.org/t/eip-3372-apply-minor-modifications-to-the-ethash-algorithm-to-break-current-asic-implementations-eip-969-resubmission/5655
+created: 2021-03-13
+---
+
+# Simple Summary
+
+Introduce 5 new FNV primes into the ethash algorithm.
+
+# Abstract
+
+This EIP is to kick current ASIC implementations out of the network to keep the Ethereum network secure and healthy by changing the `fnv` constants.
+
+# Motivation
+
+ASICs provide a severe centralization risk for the Ethereum network. If we do not get rid of them, small GPU miners will be forced to exit the Ethereum mining because EIP-1559 will make them mining at a loss. Furthermore, ASIC production will be concentrated only at one or two parties, which will make the Ethereum hashrate centralized. Also, it is worth noting that Ethash ASIC machines cost 10k+ USD, while GPUs are priced less than 1000 USD.
+
+With GPUs, miners can switch to other mining algorithms, but with ASICs, it is impossible. Leaving everything as-is will almost certainly a very tough (from the side of miners) integration of the Ethereum 2.0.
+
+In short, this EIP is required to keep the Ethereum network stable and decentralized by keeping the ASIC business away.
+
+# Specification
+
+If `block.number >= ETHASH11_BLKNUM`, activate the `ethash1.1` algorithm version.
+
+## ethash1.1
+
+Prior to this change, `fnv` hash function is used throughout the `hashimoto` function. `fnv` is identical for all steps, `ethash1.1` will introduce additional `fnvA`, `fnvB`, `fnvC`, `fnvD`, and `fnvE` functions. All those functions will have different FNV constants.
+
+```c
+// Previously used FNV prime
+#define FNV_PRIME_0 0x1000193
+
+// New FNV primes
+#define FNV_PRIME_A 0x10001a7
+#define FNV_PRIME_B 0x10001ab
+#define FNV_PRIME_C 0x10001cf
+#define FNV_PRIME_D 0x10001e3
+#define FNV_PRIME_E 0x10001f9
+```
+
+Prior to this EIP, all parts of Ethash are using the `fnv` (hereinafter referenced as `fnv0`) function.  As of the introduction of this EIP:
+* `fnvA` replaces `fnv0` in the DAG item selection step
+* `fnvB` replaces `fnv0` in the DAG item mix step
+* `fnvC(fnvD(fnvE` replaces `fnv0(fnv0(fnv0(` in the compress mix step
+* `fnv0` in the DAG generation step should remain unchanged to keep previous DAGs valid.
+
```

Comment:
```suggestion

## Rationale

```
The rationale section should describe why specific choices were made.  For example, why these particular primes were used, or why `fnv0` is unchanged.

Reply:
Rationale section is required for this to be merged as a draft.